<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>饭团君</author>
 <class>FileCodeToolDialog</class>
 <widget class="QDialog" name="FileCodeToolDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>463</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>File Code Tool</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>File Type</string>
   </property>
  </widget>
  <widget class="QComboBox" name="FileTypeCombo">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>30</y>
     <width>161</width>
     <height>22</height>
    </rect>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>*.cpp;*.h;*.hpp;*.c</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*.txt</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>*.html;*.htm</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>File Dir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="SrcDirEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>70</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>F:\work\home\zhanren\old\soft\server</string>
   </property>
   <property name="placeholderText">
    <string>Search Dir</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SrcDirBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Dir</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string>Changing Function</string>
   </property>
   <widget class="QRadioButton" name="UToARadio">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>UTF-8 To ANSI</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="AToURadio">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>ANSI To UTF-8</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QProgressBar" name="Progress">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
   <property name="textVisible">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="OKBtn">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>241</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start Change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Save Dir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="DstDirEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>120</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>F:\work\home\zhanren\old\soft\新建文件夹</string>
   </property>
   <property name="placeholderText">
    <string>Cover Src File By Empty</string>
   </property>
  </widget>
  <widget class="QPushButton" name="DstDirBtn">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>120</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Choose Dir</string>
   </property>
  </widget>
  <widget class="QLabel" name="WorkingLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>410</y>
     <width>441</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="FailListWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>401</width>
     <height>101</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>Fail List</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="FileCodeTool.qrc"/>
 </resources>
 <connections/>
</ui>
